### ISSUE TITLE
Shell integration doesn't work on windows w screen reader mode enabled

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#172287]
Discovered in #172149

**Comment by Tyriar** [comment]
Switched to feature to make sure it's mentioned in release notes

**Comment by alexr00** [comment]
I can't get shell integration working in VS Code stable (I just copied my insiders settings, but no luck) to confirm that I see the original bug, but I toggled on screen reader accessibility mode with "Toggle Screen Reader Accessibility Mode" then opened a terminal. The terminal had shell integration ✅
